README.md

```diff
@@ -56,6 +56,25 @@ npm start
 
 See: https://docs.sst.dev for more info on how things get deployed.
 
+## Package Tests
+
+To run per-package tests, first install Docker Desktop (https://www.docker.com/) and ensure it is running.
+
+Next, ensure the `AWS_REGION`, `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables are set in your terminal. They do 
+not need to be set to real values - the following works in `bash`-like shells:
+
+```
+export AWS_REGION='us-west-2'; export AWS_ACCESS_KEY_ID='NOSUCH'; export AWS_SECRET_ACCESS_KEY='NOSUCH
+```
+
+Finally, run the tests with:
+
+```
+npm test
```

Comment:
From the workspace root or the package dir?

I think with `npm` the way to run per package tests is `npm test -w <path/to/package>` from the root. Doing `npm` things from the packages often doesn't work as expected.

Reply:
yep, worth documenting the per-package test command too, I'll add!